client/post-editor/media-modal/detail/detail-preview-videopress.jsx

```diff
@@ -172,16 +164,17 @@ class EditorMediaModalDetailPreviewVideoPress extends Component {
 
 	render() {
 		const classes = classNames( this.props.className, 'is-video' );
-		const { isPlaying, item } = this.props;
+		const { item } = this.props;
 		const { height = 480, videopress_guid, width = 854 } = item;
 
 		const params = {
-			autoPlay: isPlaying,
+			autoPlay: this.autoplay,
```

Comment:
If we keep the `shouldComponentUpdate` logic we won't be needing the component instance property `autoplay` IMHO and we could just continue using `isPlaying`, no?